services/consensusalgo/leanhelixconsensus/service.go

```diff
@@ -105,6 +105,7 @@ func NewLeanHelixConsensusAlgo(
 		KeyManager:          mgr,
 		ElectionTimeoutOnV0: config.LeanHelixConsensusRoundTimeoutInterval(),
 		Logger:              NewLoggerWrapper(parentLogger, config.LeanHelixShowDebug()),
+		OnElectionCB:        s.onElection,
```

Comment:
can you add a test that would protect us if this happened again?

Reply:
Opened https://github.com/orbs-network/orbs-network-go/issues/1350